generated/storage/notifier.pb.go

```diff
@@ -3006,6 +3026,9 @@ func (m *Email) Size() (n int) {
 	if m.StartTLSAuthMethod != 0 {
 		n += 1 + sovNotifier(uint64(m.StartTLSAuthMethod))
 	}
+	if m.AllowUnauthenticatedSmtp {
+		n += 2
+	}
```

Comment:
Whatever this is, it looks frighteningly brittle.

Reply:
Wait what, I've never noticed this in generated code. Crazy. But protobuf.